ci-operator/config/openshift-knative/serving/openshift-knative-serving-release-next__48.yaml

```diff
@@ -257,25 +74,122 @@ tests:
     timeout: 1h0m0s
     version: "4.8"
   steps:
+    allow_best_effort_post_steps: true
+    post:
+    - as: knative-must-gather
+      best_effort: true
+      cli: latest
+      commands: oc adm must-gather --image=quay.io/openshift-knative/must-gather --dest-dir
+        "${ARTIFACT_DIR}/gather-knative"
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+      timeout: 20m0s
     test:
     - as: test
       cli: latest
       commands: make test-e2e
       dependencies:
       - env: KNATIVE_SERVING_ACTIVATOR
         name: knative-serving-activator
-      - env: KNATIVE_SERVING_AUTOSCALER
-        name: knative-serving-autoscaler
       - env: KNATIVE_SERVING_AUTOSCALER_HPA
         name: knative-serving-autoscaler-hpa
+      - env: KNATIVE_SERVING_AUTOSCALER
+        name: knative-serving-autoscaler
       - env: KNATIVE_SERVING_CONTROLLER
         name: knative-serving-controller
+      - env: KNATIVE_SERVING_DOMAIN_MAPPING_WEBHOOK
+        name: knative-serving-domain-mapping-webhook
       - env: KNATIVE_SERVING_DOMAIN_MAPPING
         name: knative-serving-domain-mapping
+      - env: KNATIVE_SERVING_MIGRATE
```

Comment:
All the additional dependencies should be fine, right?

Reply:
I think so. If we missed it, it will produce some kind of failure, won't it?